cookbooks/symfony_no_flex.rst

```diff
@@ -0,0 +1,300 @@
+SymfonyCloud without Flex
+=========================
+
+SymfonyCloud experience is optimal when Symfony applications are using Flex, but
+not every project has been migrated to Flex yet. Even if you are using Symfony
+2.x or 3.x, you can start using SymfonyCloud and actually leverage some of its
+features to ease the migration to Symfony 4 and Flex.
+
+.. tip::
+
+   Although this guide helps you setuping your own project for SymfonyCloud, we
+   suggest you first follow the :doc:`Getting Started </getting-started>`
+   guide with a brand new Symfony 4 project and a free trial to get a better
+   understanding of SymfonyCloud features and philosophy.
+
```

Comment:
I would add the warning tip about free trial here (instead of later, or in addition to)

Reply:
done (in a nested "caution" wrapper)